### PR TITLE
Create weekly-update.2015-05-22.md

### PR DESCRIPTION
**Pull request by Gioyik**
#73

### CODE REVIEW

File: weekly-updates/weekly-update.2015-05-22.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+* Surrogate pair in REPL can freeze terminal [#690](https://github.com/nodejs/io.js/issues/690)
+* `process.send()` is not synchronous as the docs suggest, a regression introduced in 1.0.2, see [#760](https://github.com/nodejs/io.js/issues/760) and fix in [#774](https://github.com/nodejs/io.js/issues/774)
+* Calling `dns.setServers()` while a DNS query is in progress can cause the process to crash on a failed assertion [#894](https://github.com/nodejs/io.js/issues/894)
+* `url.resolve` may transfer the auth portion of the url when resolving between two full hosts, see [#1435](https://github.com/nodejs/io.js/issues/1435).
+
+### Community Updates
+
+* Mikeal Rogers post about **Promise errors in io.js** on [Modulus.io](http://blog.modulus.io/promise-errors-in-iojs)
+* A lot of cities joined NodeSchool International Day.
+* Logjam attack vulnerability dettected on io.js. Will fixed the vulnerability on next release.
```
Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-05-22.md]: io.js v2.1.0 fixed the Logjam attack vulnerability.

Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-05-22.md]: The pull-request url is here. https://github.com/nodejs/io.js/pull/1739

Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-05-22.md]: - [Logjam](https://weakdh.org/) attack vulnerability detected on `Diffie-Hellman Key exchange` . io.js [fixed the vulnerability](https://github.com/nodejs/io.js/pull/1739) on [v2.1.0](https://github.com/nodejs/io.js/blob/master/CHANGELOG.md#2015-05-24-version-210-rvagg)

**Comment by Gioyik** [comment]
change done!

**Comment by Gioyik** [comment]
ok, I changed the text for the latest you suggested.

File: weekly-updates/weekly-update.2015-05-22.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* Some problems with unreferenced timers running during `beforeExit` are still to be resolved. See [#1264](https://github.com/nodejs/io.js/issues/1264).
+* Surrogate pair in REPL can freeze terminal [#690](https://github.com/nodejs/io.js/issues/690)
+* `process.send()` is not synchronous as the docs suggest, a regression introduced in 1.0.2, see [#760](https://github.com/nodejs/io.js/issues/760) and fix in [#774](https://github.com/nodejs/io.js/issues/774)
+* Calling `dns.setServers()` while a DNS query is in progress can cause the process to crash on a failed assertion [#894](https://github.com/nodejs/io.js/issues/894)
+* `url.resolve` may transfer the auth portion of the url when resolving between two full hosts, see [#1435](https://github.com/nodejs/io.js/issues/1435).
+
+### Community Updates
+
+* Mikeal Rogers post about **Promise errors in io.js** on [Modulus.io](http://blog.modulus.io/promise-errors-in-iojs)
+* A lot of cities joined NodeSchool International Day.
```
Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-05-22.md]: - [NodeSchool International Day](http://nodeschool.io/international-day/) has been held for the first time. [40 cities](https://github.com/nodeschool/international-day/issues?q=label%3Arollcall-2015+is%3Aclosed) joined.

**Comment by Gioyik** [comment]
@yosuke-furukawa changed! thanks a lot

**Comment by yosuke-furukawa** [comment]
yay, LGTM :D

File: weekly-updates/weekly-update.2015-05-22.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# io.js 1.8, 2.0 and 2.1 releases
+This week we had three io.js releases [v1.8.2](https://iojs.org/dist/v1.8.2/), [v2.0.2](https://iojs.org/dist/v2.0.2/) and [v2.1.0](https://iojs.org/dist/v2.10/), complete changelog can be found [on GitHub](https://github.com/iojs/io.js/blob/master/CHANGELOG.md).
```
Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-05-22.md]: oops. v2.1.0 url is wrong. https://iojs.org/dist/v2.10/ => https://iojs.org/dist/v2.1.0/

File: weekly-updates/weekly-update.2015-05-22.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+# io.js 1.8, 2.0 and 2.1 releases
+This week we had three io.js releases [v1.8.2](https://iojs.org/dist/v1.8.2/), [v2.0.2](https://iojs.org/dist/v2.0.2/) and [v2.1.0](https://iojs.org/dist/v2.1.0/), complete changelog can be found [on GitHub](https://github.com/iojs/io.js/blob/master/CHANGELOG.md).
```
Comment (diagramatics) [inline weekly-updates/weekly-update.2015-05-22.md:2]: Just to circumvent redirection, maybe change the link to the new org name. https://github.com/nodejs/io.js/blob/master/CHANGELOG.md.

File: weekly-updates/weekly-update.2015-05-22.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+* **tls**: default ciphers have been updated to use gcm and aes128 (Mike MacCana) [#1660](https://github.com/nodejs/io.js/pull/1660)
+* **build**: v8 snapshots have been re-enabled by default as suggested by the v8 team, since prior security issues have been resolved. This should give some perf improvements to both startup and vm context creation. (Trevor Norris) [#1663](https://github.com/nodejs/io.js/pull/1663)
+* **src**: fixed preload modules not working when other flags were used before `--require` (Yosuke Furukawa) [#1694](https://github.com/nodejs/io.js/pull/1694)
+* **dgram**: fixed `send()`'s callback not being asynchronous (Yosuke Furukawa) [#1313](https://github.com/nodejs/io.js/pull/1313)
+* **readline**: emitKeys now keeps buffering data until it has enough to parse. This fixes an issue with parsing split escapes. (Alex Kocharin) [#1601](https://github.com/nodejs/io.js/pull/1601)
+* **cluster**: works now properly emit 'disconnect' to `cluser.worker` (Oleg Elifantiev) [#1386](https://github.com/nodejs/io.js/pull/1386)
```
Comment (diagramatics) [inline weekly-updates/weekly-update.2015-05-22.md:22]: Typo on cluser. Should be cluster.

File: weekly-updates/weekly-update.2015-05-22.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+* **events**: uncaught errors now provide some context (Evan Lucas) [#1654](https://github.com/nodejs/io.js/pull/1654)
+
+#### 2.1.0
+* **crypto**: Diffie-Hellman key exchange (DHE) parameters (`'dhparams'`) must now be 1024 bits or longer or an error will be thrown. A warning will also be printed to the console if you supply less than 2048 bits. See https://weakdh.org/ for further context on this security concern. (Shigeki Ohtsu) [#1739](https://github.com/nodejs/io.js/pull/1739).
```
Comment (diagramatics) [inline weekly-updates/weekly-update.2015-05-22.md:26]: Nitpicking. There are two full stops in this line.

File: weekly-updates/weekly-update.2015-05-22.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+* **node**: A new `--trace-sync-io` command line flag will print a warning and a stack trace whenever a synchronous API is used. This can be used to track down synchronous calls that may be slowing down an application. (Trevor Norris) [#1707](https://github.com/nodejs/io.js/pull/1707).
```
Comment (diagramatics) [inline weekly-updates/weekly-update.2015-05-22.md:27]: Two full stops in this line as well.

File: weekly-updates/weekly-update.2015-05-22.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+* `process.send()` is not synchronous as the docs suggest, a regression introduced in 1.0.2, see [#760](https://github.com/nodejs/io.js/issues/760) and fix in [#774](https://github.com/nodejs/io.js/issues/774)
+* Calling `dns.setServers()` while a DNS query is in progress can cause the process to crash on a failed assertion [#894](https://github.com/nodejs/io.js/issues/894)
+* `url.resolve` may transfer the auth portion of the url when resolving between two full hosts, see [#1435](https://github.com/nodejs/io.js/issues/1435).
+
+### Community Updates
+
+* Mikeal Rogers post about **Promise errors in io.js** on [Modulus.io](http://blog.modulus.io/promise-errors-in-iojs)
+* [NodeSchool International Day](http://nodeschool.io/international-day/) has been held for the first time. [40 cities](https://github.com/nodeschool/international-day/issues?q=label%3Arollcall-2015+is%3Aclosed) joined.
+* [Logjam](https://weakdh.org/) attack vulnerability detected on Diffie-Hellman Key exchange . io.js [fixed the vulnerability](https://github.com/nodejs/io.js/pull/1739) on [v2.1.0](https://github.com/nodejs/io.js/blob/master/CHANGELOG.md#2015-05-24-version-210-rvagg).
```
Comment (diagramatics) [inline weekly-updates/weekly-update.2015-05-22.md:46]: > ... Diffie-Hellman Key exchange . io.js...

There's an extra space after exchange.

File: weekly-updates/weekly-update.2015-05-22.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
+# io.js 1.8, 2.0 and 2.1 releases
+This week we had three io.js releases [v1.8.2](https://iojs.org/dist/v1.8.2/), [v2.0.2](https://iojs.org/dist/v2.0.2/) and [v2.1.0](https://iojs.org/dist/v2.1.0/), complete changelog can be found [on GitHub](https://github.com/iojs/io.js/blob/master/CHANGELOG.md).
+
+### Notable changes
+
+#### 1.8.2
+
+**Maintenance release**
+
+* **crypto**: significantly reduced memory usage for TLS (Fedor Indutny & Сковорода Никита Андреевич) [#1529](https://github.com/iojs/io.js/pull/1529)
```
Comment (diagramatics) [inline weekly-updates/weekly-update.2015-05-22.md:10]: This one needs the new org name in the link as well. https://github.com/nodejs/io.js/pull/1529

**Comment by yosuke-furukawa** [comment]
ping @Gioyik

**Comment by yosuke-furukawa** [comment]
OK I will merge and I will fix the whole points

**Comment by Gioyik** [comment]
Sorry I was traveling. I arrived to home a few hours ago.

**Comment by yosuke-furukawa** [comment]
No problem :D I will do that. safe travel!!

**Pull request merged**